src/cooperation/components/EditCooperationForm.vue

```diff
@@ -231,26 +231,10 @@ import {
   CooperationContactsInterface,
   CooperationActionEnum,
   CooperationGettersEnum,
-} from '@/store/cooperation/types';
+} from '@/cooperation/store/types';
 import { mapGetters } from 'vuex';
 import useVuelidate from '@vuelidate/core';
-import {
-  requiredValidator,
-  edrpouValidator,
-  emailLastCharsValidator,
-  emailMaxLength,
-  emailMinLength,
-  emailValidator,
-  someTitleLenghtValidator,
-  ukrLangTitleValidator,
-  ibanValidator,
-  userPhoneValidator,
-  houseNumAndHouseBlockValidator,
-  zipCodeValidator,
-  regionCityDistrictMaxLength,
-  streetMaxLength,
-  houseBlockHouseNumberMaxLength,
-} from '@/utils/validators';
+import { editCooperationValidators } from '@/cooperation/utils/validators/cooperation-validations';
```

Comment:
As for me, cooperationValidations would be better name for the object, cause we can use it during adding house too, not only for editing

Reply:
changed